javascript/ql/src/Security/CWE-730/examples/server-crash.GOOD-B.js

```diff
@@ -0,0 +1,13 @@
+// ...
+express().post("/save", async (req, res) => {
+  try {
+    await fs.access(rootDir);
+  } catch (e) {
+    console.error(`Server setup is corrupted, ${rootDir} does not exist!`);
+    res.status(500);
+    res.end();
+  }
+  save(rootDir, req.query.path, req.body); // MAYBE BAD, depends on the commandline options
```

Comment:
With you current example, the server would attempt to `save()` even if it couldn't access `rootDir`. 

```suggestion
    res.end();
    return;
  }
  save(rootDir, req.query.path, req.body); // MAYBE BAD, depends on the commandline options
```